cmake/modules/FindDEAL_II_KOKKOS.cmake

```diff
@@ -26,7 +26,7 @@ set(KOKKOS_DIR "" CACHE PATH "An optional hint to a Kokkos installation")
 set_if_empty(KOKKOS_DIR "$ENV{KOKKOS_DIR}")
 
 
-if(DEAL_II_TRILINOS_WITH_KOKKOS)
+if(DEAL_II_TRILINOS_WITH_KOKKOS OR DEAL_II_PETSC_WITH_KOKKOS)
```

Comment:
```suggestion
if(DEAL_II_TRILINOS_WITH_KOKKOS)
```

Reply:
I'm ok with this change. We probably need more automation and to provide hints to `find_package` to look into PETSc makefile variables `KOKKOS_INCLUDE` and `KOKKOS_LIB` https://github.com/dealii/dealii/blob/a35e4628710c7d165aaf6e0d4ab00b6f6f1a6ef1/cmake/modules/FindDEAL_II_PETSC.cmake#L136